### PR TITLE
add Shyft network mainnet and testnet contract addresses

### PR DESCRIPTION
**Pull request by svanegmond**

### CODE REVIEW

File: src/assets/v1.3.0/compatibility_fallback_handler.json [hunk 1]
```diff
@@ -27,6 +27,8 @@
     "1285": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "1287": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "4002": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "7341": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "11437": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
```
Comment (rmeissner) [inline src/assets/v1.3.0/compatibility_fallback_handler.json:31]: Please make sure that this is ordered by chain id (numeric ascending)

**Comment by rmeissner** [comment]
Could you please provide an endpoint for both networks that allow checking the bytecode of the deployed contracts.

**Comment by svanegmond** [comment]
Hi Richard-

https://rpc.testnet.shyft.network <https://rpc.testnet.shyft.network/>
and
https://rpc.shyft.network/ <https://rpc.shyft.network/>

both speak regular json-rpc

There also a block explorer at https://bx.shyft.network <https://bx.shyft.network/> and https://bx.testnet.shyft.network <https://bx.testnet.shyft.network/>

File: src/assets/v1.3.0/sign_message_lib.json [hunk 2]
```diff
@@ -27,6 +27,8 @@
     "1285": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "1287": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
     "4002": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
+    "7341": "0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2",
```
Comment (rmeissner) [inline src/assets/v1.3.0/sign_message_lib.json:30]: This contract is not available
image

**Review by rmeissner — CHANGES_REQUESTED** [review]

File: src/assets/v1.3.0/sign_message_lib.json (hunk 2 above)
Comment (svanegmond) [reply src/assets/v1.3.0/sign_message_lib.json:30]: Fixed, it seems we'd deployed 1.3.0 (not libs0) before. :
`
deploying "SignMessageLib" (tx: 0xc1f33a6aecfa039552da89873857b826385b368a906fb02615214fed61da2d9a)...: deployed at 0xA65387F16B013cf2Af4605Ad8aA5ec25a2cbA3a2 with 311441 gas
`
https://bx.shyft.network/tx/0xc1f33a6aecfa039552da89873857b826385b368a906fb02615214fed61da2d9a/internal-transactions

File: src/assets/v1.3.0/compatibility_fallback_handler.json (hunk 1 above)
Comment (svanegmond) [reply src/assets/v1.3.0/compatibility_fallback_handler.json]: committed a fix for this

**Review by rmeissner — APPROVED** [review]

**Pull request merged**